processor/src/decoder/mod.rs

```diff
@@ -967,18 +967,23 @@ pub fn build_op_group(ops: &[Operation]) -> Felt {
 // DEBUG INFO
 // ================================================================================================
 
+pub struct IndexedAssemblyOp {
+    pub index: RowIndex,
+    pub op: AssemblyOp,
+}
```

Comment:
```suggestion
/// An [AssemblyOp] tagged with the clock cycle at which it occurred.
pub struct AssemblyOpAtClock {
    pub clk: RowIndex,
    pub op: AssemblyOp,
}
```

This is my attempt at making it clear that "index" really means "clock cycle". There's probably a better name than `AssemblyOpAtClock` though